gradle/protobuf.gradle

```diff
@@ -0,0 +1,13 @@
+apply plugin: 'com.google.protobuf'
+
+protobuf {
+    protoc {
+        artifact = 'com.google.protobuf:protoc:3.21.2'
```

Comment:
This shouldn't be necessary. GCV can recommend a version here.

This works on `protobuf-gradle-plugin` version 0.8.12, but not on 0.8.13. On 0.8.13 and newer, `generateProto` fails with:
```
Expected configuration ':atlasdb-client-protobufs:protobufToolsLocator_protoc' files to contain exactly one file, however, it contains no files.
```

If you manually inspect the `protobufToolsLocator_protoc` files, it definitely does have a single file. I think there's something funky caused by the fact that the plugin stores this as a file collection. I think the regression happened in https://github.com/google/protobuf-gradle-plugin/pull/408, but I'm not sure.

Reply:
Won't specifying our protobuf version directly here make upgrading protobuf's version harder (i.e. we'll need to update in multiple placers)?

Reply:
The version of `protoc` was already specified separately before this PR in `libVersions.protoc`. This PR just moves that from `libVersions` to here.